net-p2p/eiskaltdcpp/eiskaltdcpp-2.2.10.ebuild

```diff
@@ -0,0 +1,149 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="5"
+
+PLOCALES="be bg cs de el en es eu fr hu it pl pt_BR ru sk sr@latin sr sv_SE uk vi zh_CN"
+
+inherit cmake-utils eutils l10n fdo-mime gnome2-utils
+[[ ${PV} = *9999* ]] && inherit git-r3
+
+DESCRIPTION="Qt based client for DirectConnect and ADC protocols, based on DC++ library"
+HOMEPAGE="https://github.com/eiskaltdcpp/eiskaltdcpp"
+
+LICENSE="GPL-2 GPL-3"
+SLOT="0"
+IUSE="cli daemon dbus +dht +emoticons examples -gtk idn -javascript json libcanberra libnotify lua +minimal pcre +qt4 sound spell sqlite upnp -xmlrpc"
+
+REQUIRED_USE="
+	cli? ( ^^ ( json xmlrpc ) )
+	emoticons? ( || ( gtk qt4 ) )
+	dbus? ( qt4 )
+	javascript? ( qt4 )
+	json? ( !xmlrpc )
```

Comment:
I think it would be cleaner as:

    ?? ( json xmlrpc )

because it indicates that only one of the two can be enabled, correct?